docs/reference-guides/backup-restore-configuration/examples.md

```diff
@@ -250,17 +250,34 @@ spec:
 
 The snippet below demonstrates two different types of secrets and their relevance with respect to Backup and Restore of custom resources.
```

Comment:
I think it might be better "flow" if the part about creating the secret was here.
So both "Creating this secret can be done with the following command:" and the example command you added here.